src/Integration.Vsix/Manifests/VS2017/source.extension.vsixmanifest

```diff
@@ -33,6 +33,7 @@
         <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="File" Path="Google.Protobuf.dll" />
         <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="File" Path="Google.Protobuf.dll" />
         <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="Integration.TeamExplorer" Path="|Integration.TeamExplorer|" />
+        <Asset Type="Microsoft.VisualStudio.MefComponent" Path="SonarLint.VisualStudio.IssueVisualization.dll" />
```

Comment:
Out of curiosity, why is it different from the way we declare other project assets? (e.g. Integration.TeamExplorer)

Reply:
That's weird: I thought I'd cut and paste the setting from the VS2019 manifest. I'll see what happens if I change it.

Reply:
I cut and paste the entry from the built vsix, which has translated the "Project" etc settings into a "Path" to a file in the vsix. I'll change the other manifests to be the same as the VS2019 one.